libprecompiled/WorkingSealerManagerImpl.cpp

```diff
@@ -68,8 +68,19 @@ void WorkingSealerManagerImpl::rotateWorkingSealer()
     {
         return;
     }
-    // check VRFInfos firstly
-    checkVRFInfos();
+    try
+    {
+        // check VRFInfos firstly
+        checkVRFInfos();
+    }
+    catch (PrecompiledException& e)
+    {
+        PRECOMPILED_LOG(WARNING) << LOG_DESC("rotateWorkingSealer failed for checkVRFInfos failed")
+                                 << LOG_DESC("notifyNextLeaderRotate now");
+        setSystemConfigByKey(
+            INTERNAL_SYSTEM_KEY_NOTIFY_ROTATE, "1", m_context->blockInfo().number + 1);
+        throw e;
```

Comment:
Codacy found an issue: [Throwing a copy of the caught exception instead of rethrowing the original exception.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5795688)